client/extensions/woocommerce/app/products/product-form-simple-card.js

```diff
@@ -0,0 +1,155 @@
+import FormTextInput from 'components/forms/form-text-input';
+import FormTextInputWithAffixes from 'components/forms/form-text-input-with-affixes';
+
+const ProductFormSimpleCard = ( { product, editProduct, translate } ) => {
+	const setDimension = ( e ) => {
+		const dimensions = { ...product.dimensions, [ e.target.name ]: e.target.value };
+		editProduct( product, { dimensions } );
+	};
+
+	const setWeight = ( e ) => {
+		const weight = e.target.value;
+		Number( weight ) >= 0 && editProduct( product, { weight } );
+	};
+
+	const setPrice = ( e ) => {
+		editProduct( product, { regular_price: e.target.value } );
+	};
+
+	const toggleStock = () => {
+		editProduct( product, { manage_stock: ! product.manage_stock } );
+	};
+
+	const setStockQuantity = ( e ) => {
+		const stock_quantity = e.target.value;
+		Number( stock_quantity ) >= 0 && editProduct( product, { stock_quantity } );
+	};
+
+	const setBackorders = ( e ) => {
+		editProduct( product, { backorders: e.target.value } );
+	};
+
+	// TODO Pull in currency and currency position.
+	const renderPrice = () => (
+		<Card className="products__product-form-price">
+			<FormLabel>{ translate( 'Price' ) }</FormLabel>
+			<FormCurrencyInput
+				currencySymbolPrefix="$"
+				name="price"
+				value={ product.regular_price || '' }
+				onChange={ setPrice }
+				size="4"
+			/>
+		</Card>
+	);
+
+	// TODO pull in dimension and weight units.
+	const renderDeliveryDetails = () => (
+		<Card className="products__product-form-delivery-details">
+			<div className="products__product-dimensions-weight">
+				<FormFieldSet className="products__product-dimensions-input">
+					<FormLabel>{ translate( 'Dimensions' ) }</FormLabel>
+					<FormDimensionsInput
+						unit={ translate( 'in' ) }
+						dimensions={ product.dimensions }
+						onChange={ setDimension }
+					/>
+				</FormFieldSet>
+				<FormFieldSet className="products__product-weight-input">
+					<FormLabel>{ translate( 'Weight' ) }</FormLabel>
+					<FormTextInputWithAffixes
+						name="weight"
+						type="number"
+						min="0"
+						suffix="g"
+						value={ product.weight || '' }
+						onChange={ setWeight }
+						size="4"
+					/>
+				</FormFieldSet>
+			</div>
+			<FormSettingExplanation>{ translate(
+				'Shipping services will use this data to provide accurate rates.'
+			) }</FormSettingExplanation>
+		</Card>
+	);
+
+	const renderStock = () => (
+		<Card className="products__product-form-stock">
+			<div className="products__product-manage-stock-wrapper">
+				<FormLabel>{ translate( 'Manage stock' ) }</FormLabel>
+				<div className="products__product-manage-stock">
+					<div className="products__product-manage-stock-checkbox">
+						<FormCheckbox
+							checked={ Boolean( product.manage_stock ) }
+							name="manage_stock"
+							onChange={ toggleStock } />
+					</div>
+
+					{ product.manage_stock && (
+						<FormTextInput
+							name="stock_quantity"
+							value={ product.stock_quantity || '' }
```

Comment:
@coderkevin Yeah it should have. However, I already have a `/try` branch open that changes the behavior of the stock inputs a bit based on design feedback.